cli/lib/exec/open.js

```diff
@@ -27,42 +35,27 @@ const processOpenOptions = (options = {}) => {
 
   const args = []
 
-  if (options.config) {
-    args.push('--config', options.config)
-  }
+  const optionList = [
```

Comment:
Instead of listing the options here, suggest adding another property to the option object that indicates if it's used in `open` mode, `run` mode, or both modes. Then you could just query for the options that apply to the particular mode.